# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/angletypes.inl
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/angletypes.inl
--- a/gfx/angle/checkout/src/libANGLE/angletypes.inl	Sat Sep 07 13:36:46 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,35 +0,0 @@
-//
-// Copyright (c) 2012-2015 The ANGLE Project Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-//
-
-// angletypes.inl : Inline definitions of some functions from angletypes.h
-
-namespace gl
-{
-
-inline bool operator==(const Rectangle &a, const Rectangle &b)
-{
-    return a.x == b.x &&
-           a.y == b.y &&
-           a.width == b.width &&
-           a.height == b.height;
-}
-
-inline bool operator!=(const Rectangle &a, const Rectangle &b)
-{
-    return !(a == b);
-}
-
-inline bool operator==(const SamplerState &a, const SamplerState &b)
-{
-    return memcmp(&a, &b, sizeof(SamplerState)) == 0;
-}
-
-inline bool operator!=(const SamplerState &a, const SamplerState &b)
-{
-    return !(a == b);
-}
-
-}  // namespace gl